### PR TITLE
User8 page filter by duration

### PR DESCRIPTION
**Pull request by EmilieM71**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,17 +1,71 @@
...
+    const xend = Number(endTime.substr(0, 2)) * 60 + Number(endTime.substr(3));
+    if (xstart > xend) {
+      this.setState({
+        runtime: 1440 - xstart + xend,
+      });
+    }
+    this.setState({
+      runtime: xend - xstart,
+    });
+  }
```
Comment (ValerieVT) [inline src/App.jsx:39]: It should be useful to add a warning if runtime is < 1-2 min

File: src/components/filterByDuration/FilterByDuration.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        >
+          De :
+        </InputFormField>
+        <InputFormField name="endTime" value={endTime} onChange={handleChange}>
+          à :
+        </InputFormField>
+
+        <ButtonFormField onClick={this.handleSubmit} />
+      </form>
+    );
```
Comment (ValerieVT) [inline src/components/filterByDuration/FilterByDuration.jsx]: Some users won't have time restaint, so we should add a button "Peu importe !" or "J'ai tout mon temps !" or something like that.

File: src/App.jsx [hunk 3]
```diff
@@ -1,17 +1,71 @@
...
+      },
+      this.calculruntime,
+    );
+  }
+
+  render() {
+    const { startTime, endTime, runtime } = this.state;
+    return (
+      <div className="App">
+        <h1>entre amis</h1>
```
Comment (ValerieVT) [inline src/App.jsx]: This page is not just a filter for friends-movie-party, so can you please rename this onto "Je planifie ma soirée" or something like that ?

Comment (ValerieVT) [inline src/App.jsx]: I think h1 should be in the common header, so this title should be in h2 tags

File: src/components/filterByDuration/ButtonFormField.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function ButtonFormField() {
+  return (
+    <div className="form-group">
+      <Link to="/" className="btn">
```
Comment (ValerieVT) [inline src/components/filterByDuration/ButtonFormField.jsx]: Please add a title attribute on this button, to improve accessibility

**Review by ValerieVT — CHANGES_REQUESTED** [review]

File: src/components/filterByDuration/ButtonFormField.jsx (hunk 4 above)
Comment (EmilieM71) [reply src/components/filterByDuration/ButtonFormField.jsx]: Resolve in commit 24dc47fd5508dbeec763700c20b1df2b75b8a31c
Adding a title to the button in the ButtonFromField.jsx file

File: src/App.jsx (hunk 3 above)
Comment (EmilieM71) [reply src/App.jsx]: Resolve in commit 50ff2755cccb3cb1722240c9e304eaf36efc0a91
Replace text 'Entre amis' by 'je planifie ma soirée'

File: src/components/filterByDuration/FilterByDuration.jsx (hunk 2 above)
Comment (ValerieVT) [reply src/components/filterByDuration/FilterByDuration.jsx]: Ok for the next sprint

File: src/App.jsx (hunk 3 above)
Comment (ValerieVT) [reply src/App.jsx]: OK for the next sprint

**Review by ValerieVT — APPROVED** [review]

**Pull request merged**